pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[tool.poetry]
```

Comment:
This is new to me - does `poetry` need to be mentioned as the first line?

Reply:
Yes, this is the template generated from `poetry init`